examples/pytorch/gat/main.py

```diff
@@ -10,28 +10,11 @@
 from deepgnn.pytorch.common.dataset import TorchDeepGNNDataset
```

Comment:
Deltas on this file don't matter, it is the same as it was before. This PR is for example of migrate script not final examples.